mesheryctl/internal/cli/root/system/logs.go

```diff
@@ -41,7 +41,8 @@ import (
 var logsCmd = &cobra.Command{
 	Use:   "logs",
 	Short: "Print logs",
-	Long:  `Print history of Meshery's logs and begin tailing them.`,
+	Long:  `Print history of Meshery's logs and begin tailing them.
+	To view the logs of a specific component e.g. mesheryctl system logs meshery-istio`,
```

Comment:
@navendu-pottekkat Should we specify that this feature works for only the Kubernetes platform?

Reply:
IMO, we should as it will provide proper clarity to the user.

Reply:
We should add tailing support for Kubernetes 😄 

@muskankhedia if the contributor is not working on this issue, could you follow up and take it up? https://github.com/layer5io/meshery/issues/3234

Reply:
@navendu-pottekkat Yes sure. I think we should wait for a day for @divyanshugit reply.

Reply:
@navendu-pottekkat I am in the middle of solving this issue. Delay is caused by some electricity issues in my locality.

Reply:
> We should add tailing support for Kubernetes 😄

@navendu-pottekkat the logs and tailing feature is used both for the Docker and Kubernetes platform but the <component-name> support is only for the Kubernetes platform. So specifying the support globally for Kubernetes only could go wrong. Please correct me if I am wrong.

Reply:
If this PR solves the issue. Then I will look into another issue.

Reply:
The logs should be "tailable" as well as "detached" for both docker and kubernetes ideally. For docker, it is handled by docker-compose installed on the user's machine. So instead of modifying the docs, let's make sure that we have a consistent behaviour across both the platforms.

Reply:
I will start a discussion on this and we can decide the best route.

Reply:
> So instead of modifying the docs, let's make sure that we have a consistent behaviour across both the platforms.

In that case, should I work on this PR after the implementation is done?

Reply:
Yes, that would be most helpful.